src/plugin.js

```diff
@@ -214,9 +214,11 @@ const onPlayerReady = (player, emeError) => {
   setupSessions(player);
 
   if (window.WebKitMediaKeys) {
-    // Support Safari EME with FairPlay
-    // (also used in early Chrome or Chrome with EME disabled flag)
-    player.tech_.el_.addEventListener('webkitneedkey', (event) => {
+    player.eme.webkitneedkey = {
+      first: false,
```

Comment:
might be better to invert this boolean or maybe even rename the property to something like `seenFirstEvent`, which is more representative of what actually happened.